src/NotoSansMarchen/NotoSansMarchen-Regular.ufo/features.fea

```diff
@@ -721,14 +764,11 @@ feature pres {
-    , # Liga
-    [u_marc e_marc o_marc anusvara_marc candrabindu_marc candrabindu_marc.narrow ka_marc.sub kha_marc.sub ga_marc.sub nga_marc.sub ca_marc.sub cha_marc.sub ja_marc.sub nya_marc.sub ta_marc.sub tha_marc.sub da_marc.sub na_marc.sub pa_marc.sub pha_marc.sub ba_marc.sub ma_marc.sub tsa_marc.sub tsha_marc.sub dza_marc.sub wa_marc.sub zha_marc.sub za_marc.sub aa_marc.sub ra_marc.sub la_marc.sub sha_marc.sub sa_marc.sub ha_marc.sub a_marc.sub], # Mark
-    ;
-} GDEF;
+feature dist {
+lookup AddDist {
+	pos @BASES' lookup AddYaSubWidth ya_marc.sub;
```

Comment:
Why is the width added only after base letters? Doesn’t U+11CA9 take up the same width regardless of context?

Reply:
Yes, this is really only a quick fix to address https://github.com/googlefonts/noto-fonts/issues/1725. It would require some more work to cover everything and to do it well and even then are any of those combinations realistic for Marchen? A lot are already covered by ligatures. I would like to do all of this more thoroughly in this font and others, but I don't have the time these days.